identity-hub-cli/src/test/java/org/eclipse/edc/identityhub/cli/VerifiableCredentialsCommandTest.java

```diff
@@ -116,10 +116,10 @@ void add() throws Exception {
         // assert JWT signature
         assertThat(verifyVerifiableCredentialSignature(signedJwt)).isTrue();
 
-        var result = envelope.toVerifiableCredential(MAPPER);
+        var result = envelope.toVerifiableCredentials(MAPPER);
 
         assertThat(result.succeeded()).isTrue();
-        assertThat(result.getContent().getItem()).usingRecursiveComparison()
+        assertThat(result.getContent().get(0).getItem()).usingRecursiveComparison()
```

Comment:
See comment above : assert array size before checking the element(s)

Reply:
done